.github/ISSUE_TEMPLATE/20-feature_request.yaml

```diff
@@ -0,0 +1,65 @@
+name: Feature
+description: |
+    Suggest a new feature for next Bareos
+    (**NOTE**: we only making critical & bug fixes to Bareos current)
```

Comment:
the **NOTE** doesn't render correctly.
Maybe we can just state "Suggest a new feature for a future version of Bareos"

Reply:
Yes your sentence make sense, especially we're not sure it will be in the next or next next ;-)